user_docs/glossary.md

```diff
@@ -34,6 +34,9 @@ general:Data Steward
 general:Data Submitters 
 :   Users who are depositing data with GHGA Central (and includes the Data Controller, if not the same person(s), as defined in the Data Processing Contract). The Data Controllers will regularly operate one or several Data Access Committees (DACs) facilitating decision-making regarding access to the research data shared via the GHGA Data Infrastructure.
 
+general:Data Transfer Agreement (DTA)
+:   An agreement signed between a Research Data Controller and a Data Requester before a Data Access Request is approved. GHGA is not involved in the contractual relationship here, but is informed by the Research Data Controller once an appropriate agreement is in place. Please note: Depending on the institutions, this is sometimes also referred to as a Data Access Agreement (DAA). 
```

Comment:
Would like to add: ...also referred to as a Data Access Agreement (DAA) or Data Use Agreement (DUA).

Reply:
Good point, included in https://github.com/ghga-de/docs.ghga.de/pull/37/commits/60743fc6864e543f54619d09dc9532ebeeddbbc6